exe-unit/runtime-api/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ya-runtime-api"
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
As far as I know, we use `cargo release` to bump versions. You should probably use use patch in top level cargo.toml